### DISCUSSION TITLE
Up to date browser

### DISCUSSION THREAD

**Discussion by ankoafterhours** [RedGreenBlue09/WFAv7_Installer#20] [Q&A]
Hi, is there a way to get up to date browser? I was trying to install firefox from the windows store but it is not possible. Unfortunately some websites no longer support the version of default edge browser that is already installed.

**Comment by RedGreenBlue09** [comment]
As said in the Wiki, only ARM (ARMv7) programs runs on this build. Most browsers like Firefox and Chromium-based browsers don't have ARM support. The other ARM one I know is [Otter](https://forum.xda-developers.com/t/app-beta-otter-browser-for-windows-on-arm32.4267593/).

**Comment by RedGreenBlue09** [comment]
There has been a Chromium port in the OpenRT discord. It is half-broken, but that's some progress.
